src/stores/AutoRageshakeStore.ts

```diff
@@ -109,14 +115,19 @@ export default class AutoRageshakeStore extends AsyncStoreWithClient<IState> {
 
     private async onDeviceMessage(ev: MatrixEvent): Promise<void> {
         if (ev.getType() !== AUTO_RS_REQUEST) return;
+        const messageContent = ev.getContent();
+        const recipientRageshake = messageContent["recipient_rageshake"] || "";
         const now = new Date().getTime();
         if (now - this.state.lastRageshakeTime > RAGESHAKE_INTERVAL) {
             await this.updateState({ lastRageshakeTime: now });
             await sendBugReport(SdkConfig.get().bug_report_endpoint_url, {
-                userText: "Auto-reporting decryption error (sender)",
+                userText: `Auto-reporting decryption error (sender) ${recipientRageshake}`,
                 sendLogs: true,
                 label: "Z-UISI",
-                customFields: new Map([["matching_issue", ev.getContent().matching_issue || ""]]),
+                customFields: {
+                    "recipient_rageshake": recipientRageshake,
+                    "auto-uisi": JSON.stringify(messageContent),
```

Comment:
please stay consistent with snake case.

Reply:
This is also to match https://github.com/vector-im/element-android/pull/4325/files#diff-cae90a0eefe5f8f59e83ffbbfba49b81922cee224803331c99167f4375505bf9R228